pyrasterframes/python/setup.py

```diff
@@ -63,67 +70,39 @@ def run(self):
                 print(traceback.format_exc())
 
 
-class ZipCommand(distutils.cmd.Command):
-    """A custom command to create a minimal zip distribution."""
-
-    description = 'create a minimal pyrasterframes source zip file'
-    user_options = []
-
-    def initialize_options(self):
-        pass
-
-    def finalize_options(self):
-        pass
-
-    def run(self):
-        """Create the zip."""
-        import zipfile
-        from pathlib import Path
-        import os
-        zfile = 'pyrasterframes.zip'
-
-        if os.path.isfile(zfile):
-            os.remove(zfile)
-        with zipfile.ZipFile(zfile, 'w') as przip:
-            przip.write('pyrasterframes')
-            # Bring in source files and readme
-            patterns = ['*.py', '*.rst', '*.jar']
-            root = Path('.').resolve()
-            for pattern in patterns:
-                for file in list(root.glob('pyrasterframes/' + pattern)):
-                    przip.write(str(file.relative_to(root)))
-            # Put a copy of the license in the zip
-            przip.write('LICENSE.md', 'pyrasterframes/LICENSE.md')
-
-
-with open('README.rst') as f:
-    readme = f.read()
-
-with open('requirements.txt') as f:
-    requirements = f.read().splitlines()
-
-setup_args = dict(
+setup(
     name='pyrasterframes',
-    description='Python bindings for RasterFrames',
+    description='RasterFrames for PySpark',
     long_description=readme,
+    long_description_content_type='text/x-rst',
     version=environ.get('RASTERFRAMES_VERSION', 'SNAPSHOT'),
```

Comment:
The use of SNAPSHOT generates a Warning with `python setup.py sdist`, see below. Based on the cited [PEP440](https://www.python.org/dev/peps/pep-0440/#pre-releases) the convention would be append `'dev'` or even e.g. `'rc4.dev3'`  

```
/Users/jbrown/.local/share/virtualenvs/raster-frames-1qIqCsbO/lib/python2.7/site-packages/setuptools/dist.py:481: UserWarning: The version specified ('SNAPSHOT') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
  "details." % self.metadata.version
```

Reply:
Oh I missed what is happening here. Is the sbt build supposed to export this environment variable?

Reply:
Yes.

https://github.com/locationtech/rasterframes/pull/125/files#diff-4e22d86af425881928d88dc8c6aecf44R74

However, I'll change the default version to `dev`.